docs/reference/content/getting-started/quick-tour.md

```diff
@@ -0,0 +1,206 @@
++++
+date = "2015-03-17T15:36:56Z"
+title = "Quick Tour"
+[menu.main]
+  parent = "Getting Started"
+  identifier = "Quick Tour"
+  weight = 2
+  pre = "<i class='fa'></i>"
++++
+
+# Quick Tour
+
+Morphia wraps the mongodb Java driver so some level of familiarity with using the driver can be helpful.  Morphia does its best to 
+abstract much of that away but if something is confusing it wouldn't hurt to consult the Java driver [documentation](http://mongodb.github
+.io/mongo-java-driver/) as well.
+
+The following code snippets come from the `QuickTour.java` example code
+that can be found with the [Morphia source]({{< srcref "morphia/src/test/examples/java/org/mongodb/morphia/examples/QuickTour.java">}}).
+
+## Setting up Morphia
+
+The following example shows how to create the initial Morphia instance.  Using this instance, you can configure various aspects of how
+Morphia maps your entities and validates your queries.
+
+
+```java
+final Morphia morphia = new Morphia();
+
+// tell Morphia where to find your classes
+// can be called multiple times with different packages or classes
+morphia.mapPackage("org.mongodb.morphia.example");
+
+// create the Datastore connecting to the database running on the default port on the local host
+final Datastore datastore = morphia.createDatastore(new MongoClient(), "morphia_example");
+datastore.ensureIndexes();
+```
+
+This snippet creates the Morphia instance we'll be using in our simple application.  The `Morphia` class exists to configure the `Mapper`
+ to be used and to define various system-wide defaults.  It is also what is used to create the `Datastore` we'll be using.  With this 
+ approach we could conceivable configure Morphia once and then connect to multiple databases by creating different `Datastore` instances.
+   In practice, this is likely pretty rare but it is possible.  The `Datastore` takes two parameters:  the `MongoClient` used to connect 
+   to mongodb and the name of the database to use.
```

Comment:
I just rearranged that paragraph a little.  Does that read any better?

*--------------------------------*

*{ *name     : *"**Justin Lee**", *
  title    : *"*Software Engineer*"*,
  twitter  : *"*@evanchooly <http://twitter.com/evanchooly>*"*,
  web      : *[* "10gen.com", *"*antwerkz.com*"* *]*,
  location : *"*New York, NY*"* *}*

On Fri, May 15, 2015 at 1:33 PM, Jeff Yemin <notifications@github.com>
wrote:

> In docs/reference/content/getting-started/quick-tour.md
> <https://github.com/mongodb/morphia/pull/770#discussion_r30428496>:
>
> > +final Morphia morphia = new Morphia();
> > +
> > +// tell Morphia where to find your classes
> > +// can be called multiple times with different packages or classes
> > +morphia.mapPackage("org.mongodb.morphia.example");
> > +
> > +// create the Datastore connecting to the database running on the default port on the local host
> > +final Datastore datastore = morphia.createDatastore(new MongoClient(), "morphia_example");
> > +datastore.ensureIndexes();
> > +```
> > +
> > +This snippet creates the Morphia instance we'll be using in our simple application.  The `Morphia` class exists to configure the `Mapper`
> > + to be used and to define various system-wide defaults.  It is also what is used to create the `Datastore` we'll be using.  With this
> > + approach we could conceivable configure Morphia once and then connect to multiple databases by creating different `Datastore` instances.
> > +   In practice, this is likely pretty rare but it is possible.  The `Datastore` takes two parameters:  the `MongoClient` used to connect
> > +   to mongodb and the name of the database to use.
>
> The Datastore takes two parameters: the MongoClient used to connect to
> mongodb and the name of the database to use.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mongodb/morphia/pull/770/files#r30428496>.
>